### ISSUE TITLE
runtime error without crash

### ISSUE THREAD

**Issue by psychocoderHPC** [ComputationalRadiationPhysics/picongpu#4393]
PIConGPU is currently showing a runtime error but is not crashing.

```
picongpu/thirdParty/cupla/alpaka/include/alpaka/mem/buf/BufUniformCudaHipRt.hpp(324) 'TApi::hostFree(ptr)' returned error  : 'cudaErrorInvalidValue': 'invalid argument'!
```

This error an alpaka error https://github.com/alpaka-group/alpaka/pull/1868 and was introduced with the latest update to cupla https://github.com/ComputationalRadiationPhysics/picongpu/pull/4309. Cupla is allocating host memory for CUDA now always as mapped memory which is resulting in the error because of the strange behavior of CUDA.

**Comment by PrometheusPi** [comment]
@psychocoderHPC So does PIConGPU just continues running despite this error?

**Comment by psychocoderHPC** [comment]
Yes, PIConGPU is still running. There is a very tiny possibility that the pointer address points to a memory location PIConGPU owns and is using and that this location is freed. The change is nearly zero but sometimes you also win the jackpot in Lotto.

**Comment by steindev** [comment]
Fixed w/ #4417.
